data/battle_scripts_1.s

```diff
@@ -8072,7 +8072,8 @@ BattleScript_MoveStatDrain::
 	printstring STRINGID_TARGETABILITYSTATRAISE
 	waitmessage B_WAIT_TIME_LONG
 	clearsemiinvulnerablebit
-	tryfaintmon BS_ATTACKER, FALSE, NULL
+	tryfaintmon BS_ATTACKER, FALSE, NULL @ this should be removed, unless I'm missing something
```

Comment:
This is here just for Explosion/Self Destruct, which can be changed to electric type and absorbed. I think running `tryfaintmon` only for those move effects would be fine, but it does need to be there or they'll glitch out.

Reply:
Interesting!
Why is it not there for volt absorb then?

Reply:
For whatever reason it was working properly with Volt Absorb IIRC, but not Lightningrod/Motor Drive.

Reply:
Added the fix for waterabsorb-like abilities, also left a comment to not wonder what's up in the future